### PR TITLE
Remove annotations from comments in all source files

### PR DESCRIPTION
**Pull request by smillst**
Removes all annotations in comments from all source files.  Also, adds dependency on checker-qual for all projects.

### CODE REVIEW

File: framework/src/org/checkerframework/framework/test/TestConfigurationBuilder.java [hunk 1]
```diff
@@ -404,15 +404,7 @@ public static String getDefaultClassPath() {
...
-        return "build"
-                + File.pathSeparator
-                + "junit-4.12.jar"
-                + File.pathSeparator
-                + "hamcrest-core-1.3.jar"
-                + File.pathSeparator
-                + classpath
-                + File.pathSeparator
-                + globalclasspath;
+        return classpath + File.pathSeparator + globalclasspath;
```
Comment (mernst) [inline framework/src/org/checkerframework/framework/test/TestConfigurationBuilder.java]: Is this related to the annotations-in-comments change?

**Comment by smillst** [comment]
@wmdietl Do you want to take a look or should I merge this?
